Explorer/Assets/DCL/SDKEntityTriggerArea/SDKEntityTriggerArea.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DCL.SDKEntityTriggerArea
+{
+    public class SDKEntityTriggerArea : MonoBehaviour, IDisposable
+    {
+        [field: SerializeField] public BoxCollider BoxCollider { get; private set; }
+        [field: SerializeField] public SphereCollider SphereCollider { get; private set; }
+
+        internal readonly HashSet<Collider> currentEntitiesInside = new ();
+        internal readonly HashSet<Collider> enteredEntitiesToBeProcessed = new ();
+        internal readonly HashSet<Collider> exitedEntitiesToBeProcessed = new ();
+        [NonSerialized] public Transform TargetTransform;
```

Comment:
Define it as nullable for clarity

Reply:
tackled at https://github.com/decentraland/unity-explorer/pull/5244/commits/5368e4ebc06c7606f44abdd052c4cb2dcc6c1a56